charts/testkube-enterprise/profiles/values.demo.yaml

```diff
@@ -0,0 +1,535 @@
+      expirationPeriod: 0
+      # -- Should be set to `true` if MinIO is exposed through HTTPS
+      secure: false
+    sendgrid:
+      # -- Sendgrid API key
+      apiKey: ""
+      # -- Secret API key secret ref (secret must contain key SENDGRID_API_KEY) (default is `sendgrid-api-key`)
+      secretRef: ""
+    agent:
+      hide: false
+      # -- Agent host (without protocol) is used for building agent install commands (if blank, api will autogenerate it based on the value of `global.domain`)
+      host: "testkube-enterprise-api:grpcs"
+      # -- Agent port - used for building agent install commands
+      port: 80
+    oauth:
+      # -- OAuth secret ref for OAuth configuration (secret must contain keys: OAUTH_CLIENT_ID, OAUTH_CLIENT_SECRET, OAUTH_ISSUER_URL, OAUTH_REDIRECT_URI) (default is `testkube-cloud-oauth-secret`)
+      secretRef: ""
+      # -- OAuth Client ID for the configured static client in Dex
+      clientId: "testkube-enterprise"
+      # -- OAuth Client ID for the configured static client in Dex
+      clientSecret: "QWkVzs3nct6HZM5hxsPzwaZtq"
+      # -- If oauth.secretRef is empty (""), then oauth.redirectUri field will be used for the OAuth redirect URI
+      redirectUri: "http://localhost:8090/auth/callback"
+      # -- if oauth.secretRef is empty (""), then oauth.issuerUrl field will be used for the OAuth issuer URL
+      issuerUrl: ""
+    dashboardAddress: "http://localhost:8080"
+    apiAddress: "http://localhost:8090"
+    # -- S3 bucket in which to store logs & artifacts
+    outputsBucket: testkube-cloud-outputs
+  prometheus:
+    enabled: false
+  ingress:
+    className: nginx
+
+testkube-cloud-ui:
+  fullnameOverride: testkube-enterprise-ui
+  ui:
+    # -- Auth strategy to use (possible values: "" (default), "gitlab", "github"), setting to "" enables all auth strategies, if you use a custom Dex connector, set this to the id of the connector
+    authStrategy: ""
+    apiServerEndpoint: "http://localhost:8090"
+    rootRoute: "http://testkube-enterprise-ui.{{.Release.Namespace}}.svc.cluster.local:8080"
+  image:
+    repository: testkubeenterprise/testkube-enterprise-ui
+    tag: 1.9.0
+  ingress:
+    enabled: false
+    # -- Name of the TLS secret which contains the certificate files
+    tlsSecretName: testkube-enterprise-ui-tls
+  # -- Toggle whether to enable redirect Ingress which allows having a different subdomain redirecting to the actual Dashboard UI Ingress URL
+  ingressRedirect:
+    enabled: false
+
+testkube-agent:
+  enabled: true
+  testkube-dashboard:
+    enabled: false
+  mongodb:
+    enabled: false
+  testkube-api:
+    cloud:
+      url: testkube-enterprise-api.testkube.svc.cluster.local:8089 # FIXME to update to support tpl in https://github.com/kubeshop/helm-charts/blob/develop/charts/testkube-api/templates/deployment.yaml#L308
+      tls:
+        enabled: false
+      existingSecret:
+        name: testkube-default-agent-token
+        key: agent-token
```

Comment:
How is org and env ID set?
https://github.com/kubeshop/helm-charts/blob/develop/charts/testkube-api/values.yaml#L129-L132

Reply:
I implemented a feature so that this is no longer necessary. See [this PR](https://github.com/kubeshop/testkube/pull/5312) for details. TLDR is that we use the agent token to fetch any details we need directly from the control plane.